api/src/pcapi/core/finance/factories.py

```diff
@@ -264,13 +264,22 @@ class Meta:
     venue = factory.SubFactory(offerers_factories.VenueFactory)
     kind = models.IncidentType.OVERPAYMENT
     status = models.IncidentStatus.CREATED
+    details = {
+        "origin": "Demande e-mail",
+        "createdAt": datetime.datetime.utcnow(),
```

Comment:
(détail) Cela donne à penser qu'on stocke et récupère un objet `datetime`. Mais comme c'est JSON-encodé puis décodé, on récupère en fait une chaîne de caractères. Il serait donc plus juste ici de l'indiquer : 

```suggestion
        "createdAt": datetime.datetime.utcnow().isoformat(),
```